### ISSUE TITLE
KWS Keras Model Accuracy

### ISSUE THREAD

**Issue by HadeelMabrouk** [majianjia/nnom#139]
Hi,

I was wondering what is the reported accuracy for the Keras model in the KWS example. [Here](https://github.com/majianjia/nnom/tree/master/examples/keyword_spotting), the reported top-1 acc is reported to be = 83.371195%. However, [here](https://github.com/majianjia/nnom/tree/master/examples) it's reported to be around 90%. So, I was wondering if there was a typo in this part.

On the other hand, when I tried to retrain the Keras model (using the old configurations), I only got an accuracy of 81.7%. So, I was wondering what might be the reason for these inconsistencies.

TIA.

**Comment by majianjia** [comment]
Hi,
The training set might be different from time to time (such as volume, noise mixing level, epoch, selected keywords, or the model itself). It might reach around 90% back in time.
The accuracy for all 35 keywords will be low, but will increase if you only select 10. 
You can tune the model once you understand the process.

**Comment by HadeelMabrouk** [comment]
Noted with thanks.
